web/src/sdk/links/debugLink.js

```diff
@@ -0,0 +1,34 @@
+import { concat, ApolloLink } from 'apollo-link'
+import { logger } from "../logger"
+
+const timeStartLink = new ApolloLink((operation, forward) => {
+    operation.setContext({ start: new Date() });
+    return forward(operation);
+});
+
+const logTimeLink = new ApolloLink((operation, forward) => {
+    return forward(operation).map((data) => {
+        // data from a previous link
+        const time = new Date() - operation.getContext().start;
+        logger(`operation ${operation.operationName} took ${time} to complete`);
+        return data;
+    })
+});
+
+export const requestTimeLink = concat(timeStartLink, logTimeLink)
+
+export class DebugLink extends ApolloLink {
```

Comment:
Why not wrapper the real link inside of this and use the `requestTimeLink` only over here?

Reply:
Yeah.. This is something I should address good point.